### ISSUE TITLE
Startup NoClassDefFoundError

### ISSUE THREAD

**Issue by ArtelGG** [kangarko/ChatControl-Pro#828]
```
[14:20:15 INFO]: [ChatControl] Enabling ChatControl v7.9.1
[14:20:15 INFO]:   _____ _           _    _____            _             _  +
[14:20:15 INFO]:  / ____| |         | |  / ____|          | |           | |
[14:20:15 INFO]: | |    | |__   __ _| |_| |     ___  _ __ | |_ _ __ ___ | |
[14:20:15 INFO]: | |    | '_ \ / _` | __| |    / _ \| '_ \| __| '__/ _ \| |
[14:20:15 INFO]: | |____| | | | (_| | |_| |___| (_) | | | | |_| | | (_) | |
[14:20:15 INFO]:  \_____|_| |_|\__,_|\__|\_____\___/|_| |_|\__|_|  \___/|_|
[14:20:15 INFO]:
[14:20:15 INFO]: java.lang.NoClassDefFoundError: org/bukkit/Particle
[14:20:15 INFO]:        at me.kangarko.chatcontrol.final. .pRn.onEnable(SourceFile:166)
[14:20:15 INFO]:        at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:321)
[14:20:15 INFO]:        at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:332)
[14:20:15 INFO]:        at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:407)
[14:20:15 INFO]:        at org.bukkit.craftbukkit.v1_8_R3.CraftServer.loadPlugin(CraftServer.java:359)
[14:20:15 INFO]:        at org.bukkit.craftbukkit.v1_8_R3.CraftServer.enablePlugins(CraftServer.java:318)
[14:20:15 INFO]:        at net.minecraft.server.v1_8_R3.MinecraftServer.s(MinecraftServer.java:408)
[14:20:15 INFO]:        at net.minecraft.server.v1_8_R3.MinecraftServer.k(MinecraftServer.java:372)
[14:20:15 INFO]:        at net.minecraft.server.v1_8_R3.MinecraftServer.a(MinecraftServer.java:327)
[14:20:15 INFO]:        at net.minecraft.server.v1_8_R3.DedicatedServer.init(DedicatedServer.java:267)
[14:20:15 INFO]:        at net.minecraft.server.v1_8_R3.MinecraftServer.run(MinecraftServer.java:563)
[14:20:15 INFO]:        at java.lang.Thread.run(Unknown Source)
[14:20:15 INFO]: java.lang.ClassNotFoundException: org.bukkit.Particle
[14:20:15 INFO]:        at org.bukkit.plugin.java.PluginClassLoader.findClass(PluginClassLoader.java:92)
[14:20:15 INFO]:        at org.bukkit.plugin.java.PluginClassLoader.findClass(PluginClassLoader.java:87)
[14:20:15 INFO]:        at java.lang.ClassLoader.loadClass(Unknown Source)
[14:20:15 INFO]:        at java.lang.ClassLoader.loadClass(Unknown Source)
[14:20:15 INFO]:        at me.kangarko.chatcontrol.final. .pRn.onEnable(SourceFile:166)
[14:20:15 INFO]:        at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:321)
[14:20:15 INFO]:        at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:332)
[14:20:15 INFO]:        at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:407)
[14:20:15 INFO]:        at org.bukkit.craftbukkit.v1_8_R3.CraftServer.loadPlugin(CraftServer.java:359)
[14:20:15 INFO]:        at org.bukkit.craftbukkit.v1_8_R3.CraftServer.enablePlugins(CraftServer.java:318)
[14:20:15 INFO]:        at net.minecraft.server.v1_8_R3.MinecraftServer.s(MinecraftServer.java:408)
[14:20:15 INFO]:        at net.minecraft.server.v1_8_R3.MinecraftServer.k(MinecraftServer.java:372)
[14:20:15 INFO]:        at net.minecraft.server.v1_8_R3.MinecraftServer.a(MinecraftServer.java:327)
[14:20:15 INFO]:        at net.minecraft.server.v1_8_R3.DedicatedServer.init(DedicatedServer.java:267)
[14:20:15 INFO]:        at net.minecraft.server.v1_8_R3.MinecraftServer.run(MinecraftServer.java:563)
[14:20:15 INFO]:        at java.lang.Thread.run(Unknown Source)
[14:20:15 INFO]:   _   _                       _
[14:20:15 INFO]:   | | | | ___   ___  _ __  ___| |
[14:20:15 INFO]:   | |_| |/ _ \ / _ \| '_ \/ __| |
[14:20:15 INFO]:   |  _  | (_) | (_) | |_) \__ \_|
[14:20:15 INFO]:   |_| |_|\___/ \___/| .__/|___(_)
[14:20:15 INFO]:                     |_|
[14:20:15 INFO]: !-----------------------------------------------------!
[14:20:15 INFO]:  Error loading ChatControl v7.9.1, plugin is disabled!
[14:20:15 INFO]:  Running on 1.8.8-R0.1-SNAPSHOT (v1_8_R3) & Java 1.8.0_171
[14:20:15 INFO]: !-----------------------------------------------------!
[14:20:15 INFO]:  Error: org.bukkit.Particle
[14:20:15 INFO]: !-----------------------------------------------------!
[14:20:15 INFO]: [ChatControl] Disabling ChatControl v7.9.1
[14:20:15 INFO]: Plugin might not shut down property. Got NullPointerException: null
[14:20:15 WARN]: java.lang.NoClassDefFoundError: org/bukkit/Particle
[14:20:15 WARN]:        at me.kangarko.chatcontrol.final. .pRn.onDisable(SourceFile:418)
[14:20:15 WARN]:        at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:323)
[14:20:15 WARN]:        at org.bukkit.plugin.java.JavaPluginLoader.disablePlugin(JavaPluginLoader.java:360)
[14:20:15 WARN]:        at me.kangarko.chatcontrol.final. .pRn.native(SourceFile:381)
[14:20:15 WARN]:        at me.kangarko.chatcontrol.final. .pRn.onEnable(SourceFile:224)
[14:20:15 WARN]:        at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:321)
[14:20:15 WARN]:        at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:332)
[14:20:15 WARN]:        at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:407)
[14:20:15 WARN]:        at org.bukkit.craftbukkit.v1_8_R3.CraftServer.loadPlugin(CraftServer.java:359)
[14:20:15 WARN]:        at org.bukkit.craftbukkit.v1_8_R3.CraftServer.enablePlugins(CraftServer.java:318)
[14:20:15 WARN]:        at net.minecraft.server.v1_8_R3.MinecraftServer.s(MinecraftServer.java:408)
[14:20:15 WARN]:        at net.minecraft.server.v1_8_R3.MinecraftServer.k(MinecraftServer.java:372)
[14:20:15 WARN]:        at net.minecraft.server.v1_8_R3.MinecraftServer.a(MinecraftServer.java:327)
[14:20:15 WARN]:        at net.minecraft.server.v1_8_R3.DedicatedServer.init(DedicatedServer.java:267)
[14:20:15 WARN]:        at net.minecraft.server.v1_8_R3.MinecraftServer.run(MinecraftServer.java:563)
[14:20:15 WARN]:        at java.lang.Thread.run(Unknown Source)
[14:20:15 WARN]: Caused by: java.lang.ClassNotFoundException: org.bukkit.Particle
[14:20:15 WARN]:        at org.bukkit.plugin.java.PluginClassLoader.findClass(PluginClassLoader.java:92)
[14:20:15 WARN]:        at org.bukkit.plugin.java.PluginClassLoader.findClass(PluginClassLoader.java:87)
[14:20:15 WARN]:        at java.lang.ClassLoader.loadClass(Unknown Source)
[14:20:15 WARN]:        at java.lang.ClassLoader.loadClass(Unknown Source)
[14:20:15 WARN]:        ... 16 more
```

**Comment by Splashins** [comment]
Oh lol i got the same error xdd

**Comment by kangarko** [comment]
Thank you very much for reporting. This is solved in 7.9.2 rev.2 update.
